app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/KeyboardWidget.java

```diff
@@ -972,31 +973,42 @@ private void handleDone() {
         }
     }
 
-    private void handleModeChange() {
-        Keyboard current = mKeyboardView.getKeyboard();
+    private void handleShowKeyboard(Keyboard aKeyboard) {
         Keyboard alphabetic = mCurrentKeyboard.getAlphabeticKeyboard();
         Keyboard alphabetiCap = mCurrentKeyboard.getAlphabeticCapKeyboard();
```

Comment:
nit: Looks like `alphabetiCap` should be `alphabeticCap`. Can be addressed in follow up since it is preexisting.